globe-view/AnimatedArcLayer.ts

```diff
@@ -0,0 +1,51 @@
+import {ArcLayer} from '@deck.gl/layers';
```

Comment:
As this is lifted from the deck.gl repo, add a comment mentioning that. In the future it might save someone time if there is a regression